metricflow/cli/sample_dbt_models/sample_models/countries.sql

```diff
@@ -0,0 +1,3 @@
+select
+    *
+from {{source('tutorial', 'countries_seed')}}
```

Comment:
Huh, I always thought jinja required the spaces, like `{{ source(...) }}`. Is that just a readability thing?